### DISCUSSION TITLE
[NOT ISSUE] How to export a whole model, such as only one *.pth model file?

### DISCUSSION THREAD

**Discussion by Ingookey** [speechbrain/speechbrain#1632] [Q&A] [answered]
Focus on speech recognition "AISHELL (Mandarin)"

**Comment by TParcollet** [comment, marked as answer]
Hi, you can create a single module in the YAML as a Sequential, or Module List or whatever, and then give it to the checkpointer. See the asr_model variable here: https://huggingface.co/speechbrain/asr-wav2vec2-commonvoice-it/blob/main/hyperparams.yaml

**Comment by Ingookey** [comment]
> Hi, you can create a single module in the YAML as a Sequential, or Module List or whatever, and then give it to the checkpointer. See the asr_model variable here: https://huggingface.co/speechbrain/asr-wav2vec2-commonvoice-it/blob/main/hyperparams.yaml

Thanks for the reply, have a try follow your thoughts.
